scripts/Steven_Kolawole.py

```diff
@@ -0,0 +1,5 @@
+def task_2():
+  return "Hello World, this is Steven Kolawole with HNGi7 ID HNG-01749 and /
```

Comment:
i see a `/` after `and` in your statement

Reply:
That's an escape bar and it'd work well. But I will adjust it sha.